project/Dependencies.scala

```diff
@@ -26,6 +26,10 @@ object Dependencies {
     val r2dbcPool = "io.r2dbc" % "r2dbc-pool" % "1.0.1.RELEASE" // ApacheV2
     val r2dbcPostgres = "org.postgresql" % "r2dbc-postgresql" % "1.0.3.RELEASE" // ApacheV2
 
+    // we have to stick to this version for now: https://github.com/r2dbc/r2dbc-mssql/issues/276
+    // bumping to 1.0.1.RELEASE or later currently requires pool config initial-size=1 and max-size=1
+    val r2dbcSqlServer = "io.r2dbc" % "r2dbc-mssql" % "1.0.0.RELEASE" // ApacheV2
```

Comment:
should be `% Provided `

Reply:
Where does it come from if we mark it as `Provided`?

Reply:
It would be defined in the build of the user. We don't want to include dependencies to all drivers. At some point we might make the postgres driver provided as well.